blast_config/derived.c

```diff
@@ -594,5 +592,4 @@ derived_tng_t derived_list[] = {
     BITWORD("TRIGGER_XSC1", "trigger_xsc", 1, 1),
     BITWORD("TRIGGER_STATE_XSC", "trigger_xsc", 2, 6),
 
-  END_OF_DERIVED_CHANNELS
-};
+    END_OF_DERIVED_CHANNELS};
```

Comment:
As long as this is allowed by our style checker everything looks good to me

Reply:
The target containing this file was checked during a passing CI build: https://github.com/tim-balloon/TIMflight/actions/runs/4801605669/jobs/8544038607#step:5:1665

Reply:
Ah, right I forgot that this gets compiled in the checks.